omegatrade-app/frontend/src/app/components/company/manage-company/manage-company.component.html

```diff
@@ -0,0 +1,55 @@
+<app-header></app-header>
+<app-sidebar></app-sidebar>
+<div class="spinner" *ngIf="this.loader">
+  <mat-spinner></mat-spinner>
+</div>
+<main>
+  <div class="container-fluid mt-20">
+    <div class="row">
+      <div class="col-6" style="margin: auto;">
+        <span style="font-size: 18px;">
+          Company List
+        </span>
+      </div>
+      <div class="col-6 table-options">
+        <mat-form-field>
+          <input matInput (keyup)="applyFilter($event.target.value)" placeholder="Search">
```

Comment:
Suggestion:
We have have one public variable which can be ngModel for this input and we can observe the ngModelChange for that and call the function applyFilter on ngModelChange instead of keyup event

Reply:
the above suggestion has been added.